main/mariadb/APKBUILD

```diff
@@ -136,11 +152,9 @@ _client_libs() {
 	replaces="mariadb libmysqlclient"
 	depends="mariadb-common"
 	mkdir -p "$subpkgdir"/usr/lib \
-		"$subpkgdir"/usr/share/mysql \
-		|| return 1
-	mv "$pkgdir"/usr/lib/libmysqlclient.so* \
-		"$subpkgdir"/usr/lib/ \
-		|| return 1
+		"$subpkgdir"/usr/share/mysql
+	mv "$pkgdir"/usr/lib/libmariadb.so* \
+		"$subpkgdir"/usr/lib/
```

Comment:
are there no `libmysqlclient.so.*` anymore? If not, this will break everyting thats linked to libmysqlclient

Reply:
It has been replaced by Maria's own lib. I am pretty sure it makes a symlink for backwards compatibility.

Been a while since I worked with this PR, I still need to figure out the dbug problem.